dist/stat_data.py

```diff
@@ -730,6 +730,8 @@ def __init__(self, name, desc, flags=''):
     CompressStat('compress_write', 'compressed pages written'),
     CompressStat('compress_write_fail', 'page written failed to compress'),
     CompressStat('compress_write_too_small', 'page written was too small to compress'),
+    CompressStat('compress_precomp_leaf_maximum_page_size', 'pre-compressed maximum size for leaf pages', 'no_clear,no_scale,size'),
+    CompressStat('compress_precomp_intl_maximum_page_size', 'pre-compressed maximum size for internal pages', 'no_clear,no_scale,size'),
```

Comment:
It's nice in these cases if the text sorts together, even if the existing group of stats don't work that way. Could you figure out a way to re-word these so they start with the word compress, yet still make sense as a sentence?

Reply:
You could make the description a bit shorter as well by abbreviating `maximum` (I'm fickle with abbreviations, but this is a case where I think it's super obvious so OK): `compress_precomp_leaf_max_page_size`

Reply:
fef619a
I also moved them slightly since the names seem to be in alphabetical order (sort of).